### PR TITLE
Quick reference guide

### PR DESCRIPTION
**Pull request by sharynt**
Put together by Rich and Shaz based on the feedback we’ve received from our PR reviews.

Helpful to have one file we can dynamically update as our standards are defined/change.

Goal is to keep those who work remotely up to date and help new developers get on the same page :)

### CODE REVIEW

**Comment by Pushplaybang** [comment]
@sharynt @richlloydmiles really like this, relates to #5 really well, can we have a generalish folder? Maybe `quickstart`  - I'm sure more of this stuff will come up as time goes by.

**Comment by Pushplaybang** [comment]
just realised you put it in quick reference,  nice.

File: quick-reference/README.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+- do NOT mix formatting / cleanup work inside a feature PR
+- either: feature/ fix/ or release/
+- PRs can only go in once enough people have approved - no more just merging
+- No committing of autogenerated code (public, lib/vendor, node_modules etc)
+- No committing of secrets (like the config file)
+
+## Comments
+- Comment all the things. Always.
+- Leave extremely verbose comments, please explain your work as you go as much as possible, and be super explicit. Future you will thank you a lot for this.
+
```
Comment (slopboy) [inline quick-reference/README.md]: If possible, you should also try to explain why you have written the code.

File: quick-reference/README.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+- When linking to home page, can use: href="/" instead of esc_url(home_url('/'));
+
+- Plugins
+-- No more plugins unless they’re 150% needed
+-- Rather use libraries and do things ourselves - which is (a) quicker (b) safer (c) more extensible (d) unbloated
+-- No editing of 3rd party plugins (there are processes for when we need to really really make changes to 3rd party plugins but it should be extremely rare)
+
+## JS
+- use [] and not array()
+- no trailing commas
```
Comment (sharynt) [inline quick-reference/README.md]: @richlloydmiles - reason for this pls?

File: quick-reference/README.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+- Dev urls: It’s suggested to use dev.whatever.it.is - that way everything works seamlessly, and passwords are able to be used across subdomains
+
+## PRs
+- PR must be as small as possible (1 PR = 1 Feature)
+- do NOT mix formatting / cleanup work inside a feature PR
+- either: feature/ fix/ or release/
+- PRs can only go in once enough people have approved - no more just merging
+- No committing of autogenerated code (public, lib/vendor, node_modules etc)
+- No committing of secrets (like the config file)
+
```
Comment (sharynt) [inline quick-reference/README.md]: @solnona - sounds good. If green is happy with it, feel free to add it to this readme. @richlloydmiles - can you dicuss this with red team, and see if they'd like to adopt it too?

File: quick-reference/README.md (hunk 2 above)
Comment (richlloydmiles) [inline quick-reference/README.md]: @darrynten ?

**Pull request merged**